docs/user/dql/functions.rst

```diff
@@ -1934,6 +1935,14 @@ Example::
     | 3                           |
     +-----------------------------+
 
+    os> SELECT SECOND_OF_MINUTE((TIME '01:02:03'))
```

Comment:
Does it work? I think it should be 
```sql
SELECT SECOND_OF_MINUTE(TIME('01:02:03'))
```

Reply:
Fixed in 2b2f8b8b3873a049fe4c401c47a7eaaa71db154f